docker/cli
# Upgrade to buildx 0.12.2

Note: Depends on https://github.com/docker/buildx/pull/2244



**- What I did**
Bump the version of buildx.

**- How I did it**
Changed the version in the Dockerfile.

**- How to verify it**
Run `/usr/libexec/docker/cli-plugins/docker-buildx version` in the built container and observe the version number

**- Description for the changelog**

Upgrade to buildx 0.12.2

**- A picture of a cute animal (not mandatory but encouraged)**
